examples/to-do/index.es

```diff
@@ -0,0 +1,78 @@
+Element `to-do`
+
```

Comment:
i have mixed feels about this blank line... i think it works great when it's one file per element declaration, nothing else in the file. if it's inline with other content, such as in html or whatever, then i think you gotta have visual grouping of related code more.